scheme/app/DoctrineMigrations/Version20180308143534.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180308143534 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql("ALTER TABLE HuntGroups ADD preventMissedCalls int(10) unsigned NOT NULL DEFAULT '1'");
```

Comment:
This change require backwards compatibility protection.

See [this migration](https://github.com/irontec/ivozprovider/pull/412/commits/2f9f3a08ff0d88229283eac155fe7dc3af6948e8#diff-acc88ee29ce2dfaa25783caa116606afR23) for an example.